aten/src/ATen/core/jit_type.h

```diff
@@ -1182,17 +1183,32 @@ struct CAFFE2_API EnumType : public NamedType {
     return name().value();
   }
 
+  at::ArrayRef<TypePtr> containedTypes() const override {
```

Comment:
```suggestion
TypePtr containedType() const override {
```

Reply:
Could you elaborate? containedType() is an override and the declaration returns `ArrayRef`.

Reply:
Ooops, I thought it was a mistake, carry on.